api/v1alpha1/packagebundle_test.go

```diff
@@ -106,7 +106,87 @@ func TestPackageBundle_Find(t *testing.T) {
 	})
 }
 
-func TestMatches(t *testing.T) {
+func TestGetMajorMinorFromString(t *testing.T) {
+
+	t.Run("Parse from default Kubernetes version name", func(t *testing.T) {
+
+		target_version := "v1-21-1"
```

Comment:
nit: [Golang convention](https://go.dev/doc/effective_go#mixed-caps) is to use targetVersion rather than target_version.

Reply:
Fixed